.gitignore

```diff
@@ -24,3 +24,5 @@ openqa.log
 t/data/openqa/factory/hdd/00099963-hdd_image2.qcow2
 t/data/openqa/factory/hdd/hdd_image.qcow2
 t/diagram-v*.png
+assets/cache/
+assets/assetpack.db
```

Comment:
that's why we package. the package has a fixed version of this, but the git is only for developers not for deployment. Heavily different world we're acting in ;)